packages/ddp-server/livedata_server.js

```diff
@@ -1736,10 +1747,10 @@ var wrapInternalException = function (exception, context) {
   // provided a "sanitized" version with more context than 500 Internal server
   // error? Use that.
   if (exception.sanitizedError) {
-    if (exception.sanitizedError instanceof Meteor.Error)
+    if (exception.sanitizedError.isClientSafe)
```

Comment:
To your [question](https://github.com/meteor/meteor/pull/8756#issuecomment-307672509), I think just checking `.isClientSafe` (without also checking `instanceof Meteor.Error`) is fine, now that all `Meteor.Error` objects will have `.isClientSafe` set to `true`.